app/routes/author_routes.py

```diff
@@ -31,4 +33,23 @@ def get_all_authors():
     authors = db.session.scalars(query.order_by(Author.id))
     authors_response = [author.to_dict() for author in authors]
 
-    return authors_response
\ No newline at end of file
+    return authors_response
+
+@bp.post("/<author_id>/books")
+def create_book_with_author(author_id):
+    author = validate_model(Author, author_id)
+    
+    request_body = request.get_json()
+    request_body["author_id"] = author.id
+    new_book = Book.from_dict(request_body)
```

Comment:
I do see we're going to swap this over to `create_model` in the new lesson, but should we add a note to the lesson 8 card that the text should (like for book PUT) point out that could benefit from the same error handling as the vanilla book POST?

Reply:
It doesn't hurt to add a very short reminder in the text since we may not have time to add error handling in live codes.

Reply:
I've added a note about this in the description of the lesson 8 task